src/index.rs

```diff
@@ -1041,6 +1041,33 @@ impl Index {
             .unwrap_or_else(Vec::new)
     }
 
+    /// Returns all member variables of the given container excluding variables of type [`VariableType::Temp`]
+    /// and [`VariableType::Return`].
+    /// You should always prefer to use the [`Index::get_container_members`] method but for rare occasions
+    /// this method is needed, especially in the codegen. Specifically this method was introduced because for
+    /// ```ST
+    /// FUNCTION FOO : FOO
+    /// END_FUNCTION
+    ///
+    /// TYPE FOO : STRUCT
+    ///     bar : DINT;
+    /// END_TYPE
+    /// ```
+    /// [`Index::get_container_members`] returns `["foo", "bar"]` as its member variables because it can not
```

Comment:
@ghaith [Index::get_container_members] (among other references) doesn't seem to trigger linking, do you happen to know why that is the case?